python/mxnet/ndarray/__init__.py

```diff
@@ -25,3 +25,6 @@
 from .utils import load, save, zeros, empty, array
 from .sparse import _ndarray_cls
 from .ndarray import _GRAD_REQ_MAP
+
+__all__ = op.__all__ + ndarray.__all__ + contrib.__all__\
+          + linalg.__all__ + random.__all__ + sparse.__all__
```

Comment:
Are you able to do `from mxnet.ndarray import *` with this?

Reply:
For namespaces such as contrib, linalg, random, sparse, if their members are intended only to be accessed under the namespace, then you don't need to include it in the module's __all__

Reply:
You are right. I cannot do `from mxnet.ndarray import *`, since some of ops are registered under the submodules. @piiswrong I will remove `__all__` in `ndarray/__init__.py` and `symbol/__init__.py`.

Reply:
You can still keep the `__all__` list in `__init__`. The way to do it is
```python
__all__ = op.__all__ + ndarray.__all__ + ['contrib', 'linalg', 'random', 'sparse']
```

Reply:
Done. Thanks.